static/js/admin/src/ResultObjectActionMenu.js

```diff
@@ -115,11 +118,30 @@ define('ResultObjectActionMenu', [ 'jquery', 'jquery-ui', 'StringUtilities',  'A
 			var originalFile = resultObject.getDatastream("original_file");
 			items["viewFile"] = {name : "View File"
 				+ " ("+ StringUtilities.readableFileSize(originalFile['fileSize']) + ")"};
-
-			items["viewFits"] = {name: "View FITS"}
 		}
-		if ($.inArray('viewHidden', metadata.permissions) != -1) {
-			items["viewEventLog"] = {name : "View Event Log"};
+
+		if (viewHidden || isFile || /md_descriptive|tech_md/ig.test(datastreams)) {
```

Comment:
I think you can remove `tech_md` from the regex here, it should only/always be present if the object is a file (the work may list the tech_md of its primary object, but I don't think we need to link to it from the work).

Reply:
Also, I think it'd be fine to require `viewHidden` to access the `metadata` submenu at all. The only case where you can view an object in the admin UI that you don't have a staff role for is for a AdminUnit that you don't have permissions to. Seems reasonable to not offer admin access to datastreams for objects you don't have permission to.

If you opt to add this restriction, then you can ignore my other two comments to add `viewHidden` restrictions in. You'll also want to remove the check from the event log.